storage/connectors/jdbc/src/test/java/feast/storage/connectors/jdbc/writer/JdbcSnowflakeFeatureSinkTest.java

```diff
@@ -156,20 +126,17 @@ public void shouldWriteToSnowflake() throws SQLException {
             FeatureRow.newBuilder()
```

Comment:
I think there needs to be more extensive testing of both success and failure cases.  One of the big ones we need to check is how this job responds when features change after it has been started.  I'd like to see some thought put into others as well.